Sources/SwiftToolchain/Toolchain.swift

```diff
@@ -177,9 +185,11 @@ public final class Toolchain {
     guard let product = try inferDevProduct(hint: product)
     else { throw ToolchainError.noExecutableProduct }
 
-    if let package = package,
-      let jsKit = package.dependencies?.first(where: { $0.name == "JavaScriptKit" }),
-      jsKit.requirement.revision != ["c90e82f"] {
+    let package = try self.package.get()
+
+    if let jsKit = package.dependencies?.first(where: { $0.name == "JavaScriptKit" }),
+      jsKit.requirement.revision != ["c90e82f"]
+    {
```

Comment:
<!--
  1 failure:  Opening braces sh...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/SwiftToolchain/Toolchain.swift;
  Line: 192;
-->
- :no_entry_sign: Opening braces should be preceded by a single space and on the same line as the declaration. (`opening_brace`)